src/Sound.js

```diff
@@ -35,13 +35,18 @@ export default function Sound(props) {
   const { isLoggedIn, freeSound } = props;
```

Comment:
I think you might have added this file to your commit by accident. You can do the following to remove it:

```
git reset --soft HEAD~1
git add public/index.html
```

This command will undo the last commit while still retaining the changes from the last commit. Very useful!

Reply:
I'll keep this is mind for next time. Thanks!